### PR TITLE
Update all SS for v1.4 in UG

### PR DESCRIPTION
**Pull request by WeeeHung**
In this PR:
1. added new screenshots of the GUI for necessary commands (Prev GUI includes 'Hi Sarah', 'addressbook.json' and 'deprecated NRIC and Tags'
2. added HealthSync Logo at the start of UG
3. added some breaks in between sections as some feature section felt clustered
4. added patient card in glossary
5. removed all old version snapshots in repo

Preview lgtm but unable to test the webview of md files, so will have to check if the `<br>` is necessary and if Logo and other SS are centered. 

Possible Future extension to improve readability of UG: 
1. Add content page with hyperlinks to respective section (Not doing)
2. Check header css (currently there is no visible difference between `###` headers and `##` headers)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -40,7 +48,7 @@ HealthSync is a **powerful desktop application designed specifically for clinic
 
    A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
 
-   ![Ui](images/quickstartGUI.png)
```
Comment (WZWren) [inline docs/UserGuide.md:43]: Is `images/quickstartGUI.png` still used?

**Review by WZWren — COMMENTED** [review]
Good work adjusting the pictures to more updated versions! However, could you get rid of some of the images that are unused?

File: docs/UserGuide.md (hunk 1 above)
Comment (WeeeHung) [reply docs/UserGuide.md:43]: Yeah, there are still Screenshots for v1.3GUI in the repo. Do i remove all of them?

**Review by hyc17003 — COMMENTED** [review]
Good work formatting the UG for increased readability! For v1.4-screenshots/log.jpg, would changing 'overridden' to 'overwritten' be better? As overwritten is more of entering new data in the place of original data, which may be more accurate than using 'overridden' in this case.

File: docs/UserGuide.md (hunk 1 above)
Comment (WZWren) [reply docs/UserGuide.md:43]: Yes, please do!

**Comment by WeeeHung** [comment]
> Good work formatting the UG for increased readability! For v1.4-screenshots/log.jpg, would changing 'overridden' to 'overwritten' be better? As overwritten is more of entering new data in the place of original data, which may be more accurate than using 'overridden' in this case.

Will make the necessary changes!

**Comment by WeeeHung** [comment]
I will also be adding an SS on patient card under glossary to explain which fields appear in which row. (In a future commit in this PR)

**Comment by WZWren** [comment]
This should be fine. Good work clearing the unused images from the commit!

**Pull request merged**